_gsocproposals/2019/proposal_WLCGansible.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Python Components for the SIMPLE Grid Framework
```

Comment:
The 2 projects have identical titles, can you please change to be more specific to each proposal